foundation/foundation-maven-plugins/artifacts-maven-plugin/pom.xml

```diff
@@ -98,7 +98,6 @@
         <dependency>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-dependency-plugin</artifactId>
-            <version>${version.maven.dependency.plugin}</version>
```

Comment:
I: Moved to `dependencyManagement` of `aissemble-parent`, but I wasn't convinced that was the right move. Happy to either move it to `build-parent` or just add back these version tags.

Reply:
I think its fine for now. If a different version is needed in the future then we can override it then